__DEFINES/dates.dm

```diff
@@ -0,0 +1,4 @@
+//Useful for making time-limited things.
+
+#define IS_VALENTINES  time2text(world.realtime, "MM/DD") == "02/14"
+#define IS_APRILS_FOOL time2text(world.realtime, "MM/DD") == "04/01"
```

Comment:
should be IS_APRIL_FOOLS

Reply:
add in the christmas define I don't know where is